ios/SwiftUI/IntegrationDemo_SwiftUI.xcodeproj/project.pbxproj

```diff
@@ -368,7 +368,7 @@
 			repositoryURL = "https://github.com/TapResearch/TapResearch-iOS-SDK";
 			requirement = {
 				kind = exactVersion;
-				version = 3.2.5;
+				version = 3.5.0;
```

Comment:
what version is this?

Reply:
hm that should be 3.5.2 which is corrected in the commit right below this one